tests/console/ansible.pm

```diff
@@ -10,18 +10,27 @@
 #   4. Ansible playbook testing
 #   5. Ansible playbook execution
 #   6. Ansible Vault
-# Maintainer: Pavel Dostál <pdostal@suse.cz>
+# Maintainer: QE Core <qe-core@suse.de>, Pavel Dostál <pdostal@suse.cz>
 
 use warnings;
 use base "consoletest";
 use strict;
 use testapi qw(is_serial_terminal :DEFAULT);
 use serial_terminal 'select_serial_terminal';
 use utils qw(zypper_call random_string systemctl file_content_replace ensure_serialdev_permissions);
-use version_utils qw(is_sle is_opensuse is_tumbleweed is_transactional);
+use version_utils qw(is_sle is_opensuse is_tumbleweed is_transactional is_microos);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use transactional qw(trup_call check_reboot_changes);
 
+# git-core needed by ansible-galaxy
+# sudo is used by ansible to become root
+# python3-yamllint needed by ansible-test
+my $pkgs = 'ansible git-core';
+$pkgs .= ' python3-yamllint' unless is_transactional;
```

Comment:
`yamllint` is available on transactional systems, just not on ALP. So I would recommend to exclude it on ALP, not on all transactional systems.

```suggestion
$pkgs .= ' python3-yamllint' unless is_alp;
```